Mixpanel/Mixpanel.h

```diff
@@ -96,7 +96,7 @@ NS_ASSUME_NONNULL_BEGIN
  @discussion
  Setting a flush interval of 0 will turn off the flush timer.
  */
-@property (atomic) NSUInteger flushInterval;
+@property (nonatomic) NSUInteger flushInterval;
```

Comment:
this needs to be atomic. The setter of flush interval calls flush which in turn sets a new timer. If two threads enter at the same time, and both pass invalidate, we could be setting up 2 timers for flushing instead of 1. This performance improvement is redundant to problems it could potentially cause